client/jetpack-cloud/sections/agency-dashboard/sites-overview/test/utils.ts

```diff
@@ -69,6 +74,11 @@ describe( 'utils', () => {
 				type: 'site',
 				status: '',
 			},
+			boost: {
+				type: 'boost',
+				status: 'active',
+				data: 100,
```

Comment:
Not exactly what should go on this line.

I'm seeing a `Type 'number' is not assignable to type 'BoostObject'` which is most likely breaking my unit tests.

Reply:
data should be 

```
{
	desktop: 100,
	mobile: 100,
	overall: 100,
}
```